README.md

```diff
@@ -1,5 +1,7 @@
 # docker-postgis
 
+[![CI](https://github.com/azavea/docker-postgis/workflows/CI/badge.svg)](https://github.com/azavea/docker-postgis/actions?query=workflow%3ACI)
```

Comment:
We don't link to the workflow page in `docker-django` like we do in `django-ecsmanage` (which uses RST instead of markdown) but I think this link is a convenient addition.

Reply:
One thing we came across in setting up `pystac` is that if the `badge.svg` URL isn't filtered by a branch, it can reflect the build status of any build across the project (e.g., PR builds).  We added the `?branch=develop` there to ensure the badge only reflected the base branch's status.